tests-functional/tests/conftest.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+import os
+
+
+@pytest.fixture(scope='session')
+def session_capabilities(session_capabilities):
+    # add or modify session capabilities here
+    return session_capabilities
```

Comment:
You don't need this if you're not modifying it. It's in the cookiecutter as a helper.